src/shared/calculations.js

```diff
@@ -0,0 +1,77 @@
+const stdlib = require('@stdlib/stdlib/dist/stdlib-stats-base-dists-flat')
```

Comment:
```suggestion
import stdlib from "@stdlib/stdlib/dist/stdlib-stats-base-dists-flat";
```

Reply:
I think we should be consistent in either using es6 imports or commonJS require syntax

Reply:
You're absolutely right. I didn't think it would work in this case, but it does!